packages/oidc-client/src/lib/client.store.ts

```diff
@@ -180,5 +180,152 @@ export async function oidc<ActionType extends ActionTypes = ActionTypes>({
         }
       },
     },
+    user: {
+      info: async () => {
+        const state = store.getState();
+        const wellknown = wellknownSelector(wellknownUrl, state);
+
+        if (!wellknown?.userinfo_endpoint) {
+          const err = {
+            error: 'Wellknown missing userinfo endpoint',
+            type: 'wellknown_error',
+          } as AuthorizeErrorResponse;
+
+          log.error(err.error);
+
+          return err;
+        }
+
+        const tokens = (await storageClient.get()) as Record<string, string>;
```

Comment:
I think we should use the generic on the storageClient intialization, right? We added that because we only store one value per intialization so we can type this there. its non blocking but I think it's good for us to follow our api.